apps/asap-server/.eslintrc.js

```diff
@@ -14,5 +14,9 @@ module.exports = {
     'no-underscore-dangle': ['error', { allow: ['__typename'] }],
     'no-dupe-class-members': 'off',
     '@typescript-eslint/no-dupe-class-members': ['error'],
+    camelcase: [
+      'error',
+      { properties: 'never', allow: ['FetchResearchOutput.*'] },
```

Comment:
I don't think such setting exists for this rule https://eslint.org/docs/rules/camelcase - unless you're taking about some different way, like ignore linting inside the queries entirely? which would actually make a lot of sense - otherwise would have to update this list here all the time (unless i specify some sort of prefix for all generated types and use that here!). What do you think?

Reply:
i would ignore linting entirely. Its generated code who cares